docs/general-links.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_label: Links
+---
+# Links to external documentation
```

Comment:
```suggestion
# External documents
```

"Links" isn't the best way to describe this. Links are self-evident. External docs is really the significant essence.

File name should also probably be `external.md` or `external-documentation` or whatever